2024.html

```diff
@@ -70,6 +70,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://emmajuettner.com/">Emma Juettner</a></li>
             <li><a href="https://eklausmeier.goip.de/blog">Elmar Klausmeier</a></li>
             <li><a href="https://knowler.dev/">Nathan Knowler</a></li>
+            <li><a href="https://mart-e.be/">Martin Trigaux</a></li>
```

Comment:
Hello! Can you sort by family name instead (_Trigaux_ would come after _Proto_)? Thanks!

Reply:
oh ok, I thought somehow it was sorted by url. Actually changed it to the name of the blog as my full name doesn’t even appear on the website so found it a bit odd